### PR TITLE
WIP: Created wordcount component with staticQuery

### PR DESCRIPTION
**Pull request by alexmcode**
Add top 10 words in landing page

### CODE REVIEW

File: site/gatsby-site/page-creators/createWordCountsPage.js [hunk 1]
```diff
@@ -6,6 +6,17 @@ const stemmer = require('stemmer');
 
 const customStopWords = require('../constants/customStopWords');
 
+const PAGES_WITH_WORKCOUNT = [
```
Comment (smcgregor) [inline site/gatsby-site/page-creators/createWordCountsPage.js:9]: WORKCOUNT -> WORDCOUNT

File: site/gatsby-site/src/pages/landingPage.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import Layout from 'components/Layout';
+import { Wordlist } from '../pages/wordcounts';
+import { StyledHeading } from 'components/styles/Docs';
+
+const LandingPage = (props) => {
+  console.log(props);
```
Comment (smcgregor) [inline site/gatsby-site/src/pages/landingPage.js:8]: console logging.

**Comment by smcgregor** [comment]
Code it looking good. This is a work in progress so I will hold full review. When you make a new landing page, please drop in the filler text I have below. I will give you real text and links before we merge this. The leaderboards/word counts will go below the text.

```
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas faucibus lacinia magna a rutrum. Curabitur ante ligula, elementum eget metus et, varius pharetra sapien. Donec ullamcorper elit commodo ipsum venenatis consequat. Nulla vitae lorem blandit, pellentesque mauris non, pretium dui. Nullam nulla magna, blandit id lacus vel, ultrices malesuada erat. Vivamus id ligula ut mi dignissim imperdiet sed vel leo. Nam dui lacus, rhoncus eu imperdiet eget, efficitur non nisi. Etiam pellentesque dui lacinia, facilisis tellus sed, laoreet ipsum. Etiam pretium et ligula ac posuere. Aliquam ut faucibus nibh, vel maximus enim. Aliquam congue augue sit amet risus dapibus, ut pretium enim mollis.
```

**Comment by alexmcode** [comment]
@smcgregor I think this PR should be the last to merge from all #135 PRs. We merge #136 and #137 and then master into this branch in order to refactor and add everything from the original issue into the new landing page. does this sound optimal?

**Comment by smcgregor** [comment]
Yes, that sounds good, but you will need to remove the leaderboards from layout of earlier MRs until this merges. Otherwise the landing page starts with leaderboards and no context.

**Comment by smcgregor** [comment]
Closing in favor of: https://github.com/PartnershipOnAI/aiid/pull/148

**Pull request closed without merging**